CONFIG.example

```diff
@@ -12,8 +12,6 @@
 
 # Definitions needed for FPGA build
 export VIVADO = /dls_sw/FPGA/Xilinx/Vivado/$(VIVADO_VER)/settings64.sh
-export ISE = /dls_sw/FPGA/Xilinx/14.7/ISE_DS/settings64.sh
-export LM_LICENSE_FILE = 2100@diamcslicserv01.dc.diamond.ac.uk
```

Comment:
Why have these been removed?  The `LM_LICENSE_FILE` is definitely still needed...

Reply:
Xilinx ISE is no longer required by this repo.

LM_LICENCE_FILE -  I am not so bothered about this. It is actually no longer needed for the regular PandA builds using recent versions of Vivado, as webpack devices no longer require a licence file. If targeting non-webpack devices or using licenced IP, the licence server can be set in the personal CONFIG file (I have this set in my bash profile).  As CI builds in the cloud do not need to know about our local licence server, I would prefer to have no mention of licencing in the public repos at all.

Here is suggestion though, if you prefer:

`# Specify path to licence server, if required`
`# export LM_LICENCE_FILE = `

Reply:
I think it probably is worth adding just the empty `LM_LICENCE_FILE` then for those applications which will need a licence.